data_steward/tools/load_vocab.py

```diff
@@ -0,0 +1,270 @@
+    query_jobs = []
+    for src_table in src_tables:
+        schema = bq.get_table_schema(src_table.table_id)
+        destination = f'{project_id}.{dst_dataset_id}.{src_table.table_id}'
+        table = bq_client.create_table(Table(destination, schema=schema),
+                                       exists_ok=True)
+        job_config.destination = table
+        query = SELECT_TPL.render(project_id=project_id,
+                                  dataset_id=src_dataset_id,
+                                  table=src_table.table_id,
+                                  fields=schema)
+        query_job = bq_client.query(query, job_config=job_config)
+        LOGGER.info(f'table:{destination} job_id:{query_job.job_id}')
+        query_jobs.append(query_job)
+    wait_jobs(query_jobs)
+
+
+def main(project_id: str, bucket_name: str, dst_dataset_id: str):
+    """
+    Transform and load vocabulary files from Athena to a BigQuery dataset
+
+    :param project_id:
+    :param bucket_name: refers to the bucket containing vocabulary files from bundle
+    :param dst_dataset_id: final destination to load the vocabulary in BigQuery
+    """
+    bq_client = bq.get_client(project_id=project_id)
+    sandbox_dataset_id = get_sandbox_dataset_id(dst_dataset_id)
+    sandbox_dataset = bq.define_dataset(
+        project_id,
+        sandbox_dataset_id,
+        f'Vocabulary loaded from gs://{bucket_name}',
+        label_or_tag={'type': 'vocabulary'})
+    sandbox_dataset = bq_client.create_dataset(sandbox_dataset, exists_ok=True)
+    load_stage(sandbox_dataset, bq_client, bucket_name)
+    load(project_id,
+         bq_client,
+         sandbox_dataset_id,
+         dst_dataset_id,
+         overwrite_ok=True)
+
+
+def get_arg_parser() -> argparse.ArgumentParser:
+    """
+    Ex:
+    Typical usage
+
+        load_vocab -p my_project -b my_bucket -r 20200701
+
+    :return: the parser
+    """
+    argument_parser = argparse.ArgumentParser(description=__doc__)
+    argument_parser.add_argument(
+        '-p',
+        '--project_id',
+        dest='project_id',
+        action='store',
+        help='Identifies the project containing the target dataset',
+        required=True)
+    argument_parser.add_argument(
+        '-b',
+        '--bucket_name',
+        dest='bucket_name',
+        action='store',
+        help='Bucket containing CSV files downloaded from Athena',
+        required=True)
+    argument_parser.add_argument(
+        '-r',
+        '--release_date',
+        dest='release_date',
+        action='store',
+        help='Vocabulary release date in format yyyymmdd. Defaults to today',
+        required=False)
+    argument_parser.add_argument(
+        '-t',
+        '--target_dataset_id',
+        dest='target_dataset_id',
+        action='store',
+        help=
+        'Identifies the target dataset where the vocabulary is to be loaded',
+        required=False)
+    return argument_parser
+
+
+def parse_args() -> argparse.Namespace:
```

Comment:
Just wondering, is there a particular reason for having a separate function for doing`parse_args()`?

Reply:
Nope. Removed it.